packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -135,6 +135,21 @@
                             </div>
                         </div>
                     </div>
+                    <div class="oui-tile__item">
+                        <div class="oui-tile__definition">
+                            <strong
+                                class="oui-tile__term d-block"
+                                data-translate="overTheBox_serviceId"
+                            ></strong>
+                            <div class="oui-tile__description">
+                                <span
+                                    class="d-block mb-2"
+                                    data-ng-bind="$ctrl.serviceName"
```

Comment:
**nitpick:** `span` element is useless. You can apply `data-ng-bind="$ctrl.serviceName"` directly on the parent and get rid of the `span` element.